### DISCUSSION TITLE
Feature Request: Automatically Build RISC-V Releases for Standalone CLI

### DISCUSSION THREAD

**Discussion by arcanemachine** [tailwindlabs/tailwindcss#13140] [Ideas]
Is it possible to automatically build releases for the RISC-V architecture (riscv64)?

The main reason I am asking this question right now is that Scaleway recently made affordable RISC-V servers available for general access, further lowering the bar to access this up-and-coming architecture.

I feel it is only a matter of time before RISC-V builds are available. I would just like to get the ball rolling since Tailwind is a very important part of the modern web developer ecosystem.

Thanks

EDIT: Dug into this a bit. The only progress I was able to make was to upgrade `turbo` to `^1.12.5` in `package.json`. After that, I was able to run `pnpm install` but I couldn't run `pnpm run build` without getting a bunch of errors. I deleted the VPS and lost my error logs (whoops), but I was getting the error `Fallback bindings does not support filesystem access` a bunch of times.

**Comment by andreamancuso** [comment]
I need that too, as I'd like [RawPair](https://github.com/rawpair/rawpair) to also support RISC-V.

Anything I can do to help make that happen?

**Comment by arcanemachine** [comment]
I ended up making a wrapper for the Tailwind CLI so that I could finally get vanilla Elixir Phoenix working on a RISC-V system.

Here's a link to the repo (there's a reproducible binary) if anyone wants to build on it until RISC-V is properly supported by Bun:

https://github.com/arcanemachine/tailwindcss-riscv64-node-wrapper
